src/App/pages/LandingPage/Main/Schedule/CinemaSchedule.js

```diff
@@ -0,0 +1,15 @@
+import { MovieSchedule } from './MovieSchedule'
+export function CinemaSchedule() {
```

Comment:
Add an empty line after the end of the "imports" block.

Reply:
fixes